ac/ac-display-social/src/index.js

```diff
@@ -55,22 +55,26 @@ const ActivityRunner = props => {
   if (!data.includes(name)) {
     dataFn.listAppend(name);
   }
+
+  const others = uniq(data).filter(x => x !== name);
+  const hasOthers = others.length > 0;
```

Comment:
could be `const hasOthers = data.find(x => x!== name ) !== undefined`. That can have an impact on performance for large groups.